### ISSUE TITLE
Could not resolve dependency

### ISSUE THREAD

**Issue by macleysousa** [AlariCode/nestjs-rollbar#16]
I'm having a peer dependency error when using @nestjs/comon version 9.0.0.0

```
npm ERR! ERESOLVE unable to resolve dependency tree
npm ERR!
npm ERR! While resolving: apollo-back@0.0.1
npm ERR! Found: @nestjs/common@9.2.1
npm ERR! node_modules/@nestjs/common
npm ERR!   @nestjs/common@"^9.0.0" from the root project
npm ERR!
npm ERR! peer @nestjs/common@"^8.0.0" from nestjs-rollbar@1.7.0
npm ERR! node_modules/nestjs-rollbar
npm ERR!   nestjs-rollbar@"^1.7.0" from the root project
```

**Comment by Scriptoz** [comment]
Have the same issue:

```
npm ERR! code ERESOLVE
npm ERR! ERESOLVE unable to resolve dependency tree
npm ERR!
npm ERR! While resolving: lbank-trader@0.0.1
npm ERR! Found: @nestjs/common@9.3.9
npm ERR! node_modules/@nestjs/common
npm ERR!   @nestjs/common@"^9.2.1" from the root project
npm ERR!
npm ERR! Could not resolve dependency:
npm ERR! peer @nestjs/common@"^8.0.0" from nestjs-rollbar@1.7.0
npm ERR! node_modules/nestjs-rollbar
npm ERR!   nestjs-rollbar@"*" from the root project
npm ERR!
npm ERR! Fix the upstream dependency conflict, or retry
npm ERR! this command with --force, or --legacy-peer-deps
npm ERR! to accept an incorrect (and potentially broken) dependency resolution.
```

**Comment by ziazon** [comment]
this is old.  can we get this package updated or do we need to fork and create a new one?

**Comment by AlariCode** [comment]
PR will be welcomed.

**Comment by AlariCode** [comment]
Merged PR. Fixed in 2.0.0 version
